home/models.py

```diff
@@ -944,7 +945,7 @@ def clean(self):
 
 
 @receiver(pre_save, sender=ContentPage)
-def update_embedding(sender, instance, *args, **kwargs):
+def update_template_embedding(sender, instance, *args, **kwargs):
```

Comment:
We've renamed the `ContentPage` version to `update_template_embedding` and added a `WhatsAppTemplate` version called `update_embedding`. Can we change the names to something less confusing? Do we even need this for WhatsAppTemplate at all?

Reply:
Cool, for now, I renamed them to make more sense.  I don't know enough of it to make a call whether we need it or not, but I know we have a ticket in play on removing it, so will leave it for that one